src/Animals/Fox.ts

```diff
@@ -6,7 +6,12 @@ import { SmallDog } from './SmallDog';
 
 export class Fox extends Animal implements AttackHerdInterface {
   constructor() {
-    super(AnimalNames.FOX, undefined, AnimalRoles.PREDATOR);
+    super(
+      AnimalNames.FOX,
+      '/static/images/avatars/fox.png',
+      undefined,
```

Comment:
Is there anything we could do so we don't define it as undefined?

Reply:
We need to write it here because next argument is rested role, and rest has to be the last one. Solution for that would be remove the role from the constructor. But honestly I don't see any bigger sense of doing it only to avoid writing undefined. It's undefined anyway, because optional makes it undefined. But it is possible that the role argument will be removed in the future, so let's leave it as it is for now.